dbms/src/Common/TiFlashBuildInfo.cpp

```diff
@@ -51,15 +53,25 @@ std::string getUTCBuildTime()
 {
     return TIFLASH_UTC_BUILD_TIME;
 }
+// clang-format off
 std::string getEnabledFeatures()
 {
     std::vector<std::string> features
     {
 // allocator
 #if USE_JEMALLOC
-        "jemalloc",
+            "jemalloc",
 #elif USE_MIMALLOC
-        "mimalloc",
+            "mimalloc",
+#endif
+
+// OpenSSL or BoringSSL
+#if USE_INTERNAL_SSL_LIBRARY
+            "boringssl",
+#elif OPENSSL_VERSION_NUMBER < 0x1010100fL || defined(OPENSSL_NO_SM4)
```

Comment:
where is `OPENSSL_NO_SM4` defined

Reply:
It is defined in <openssl/opensslconf.h>. And I remember BoringSSL will define `OPENSSL_NO_SM4` so I simplify the logic here by avoiding use `USE_INTERNAL_SSL_LIBRARY`.